src/NerdMinerV2.ino.cpp

```diff
@@ -43,14 +43,15 @@ void alternate_screen_state() {
   // digitalWrite(TFT_BL, !screen_state);
   if (brightness == 255) {
     brightness = 0;
-  } else if (brightness == 250) {
+  } else if (brightness == 200) {
     brightness = 255;
   } else {
     brightness += 50;
```

Comment:
Using 51 as interval would save you an else if

Reply:
yep, I will change it! It's good idea!